### ISSUE TITLE
Bug with saving

### ISSUE THREAD

**Issue by u9g** [fdncred/nufmt#5]
Hi, very weird bug here. Saving the output with `nufmt ./a.nu | save -f a.nu` results in an empty file, but `nufmt ./a.nu | lines | str join "\n" | save -f ./a.nu` works fine for some reason.

**Comment by fdncred** [comment]
makes sense to me. this is how nushell handles raw streams of text. also, nufmt isn't anywhere near close to working, as you've probaby seen. we're looking for contributors to make it work right. feel free to jump in.
